metricbeat/cmd/modules.go

```diff
@@ -27,7 +27,8 @@ import (
 	"github.com/elastic/beats/libbeat/cmd"
 )
 
-func buildModulesManager(beat *beat.Beat) (cmd.ModulesManager, error) {
+// BuildModulesManager adds support for modules management to a beat
+func BuildModulesManager(beat *beat.Beat) (cmd.ModulesManager, error) {
```

Comment:
The only change made here is to make this method public so it can be used from x-pack, `cmd.ModulesManager` is the generic manager for modules subcommand.

Reply:
Yeah, I noticed this symbol becoming public here. But with it becoming public, the descriptive nature of the names used is even more important (a.k.a naming is hard).